GrpcSession.java

```diff
@@ -296,4 +296,30 @@ public void invoke(Req request, io.grpc.stub.StreamObserver<Resp> responseObserv
         METHOD_DESTROY_SESSION);
   }
 
+  public static interface CreateSession {
+    /**
+     * <pre>
+     **
+     * Provide a valid credential and receive a session which will
+     * be used for access to authenticated APIs
+     * </pre>
+     */
+    public void createSession(edu.hawaii.cfht.protos.authorization.CreateSessionRequest request,
+        io.grpc.stub.StreamObserver<edu.hawaii.cfht.protos.authorization.CreateSessionResponse> responseObserver) {
+      asyncUnimplementedUnaryCall(METHOD_CREATE_SESSION, responseObserver);
```

Comment:
This is not needed, because `SessionImplBase` already provides default impl. Alternatively, we might consider providing this with `--java8` flag passed to compiler.